monitoring/dashboard.py

```diff
@@ -525,6 +559,167 @@ def top10_errors_by_bucket(title, code):
     title="5xx : Top10 by Bucket", code='~"5.."'
 )
 
+
+quotaHealth = TimeSeries(
+    title="Quota service uptime",
+    dataSource="${DS_PROMETHEUS}",
+    lineInterpolation="stepAfter",
+    fillOpacity=30,
+    unit=UNITS.PERCENT_FORMAT,
+    targets=[Target(
+        expr="\n".join([
+            'sum(rate(s3_cloudserver_utilization_service_state{namespace="${namespace}", state="healthy"}[$__rate_interval])) * 100',  # noqa: E501
+            "   /",
+            'sum(rate(s3_cloudserver_utilization_service_state{namespace="${namespace}", state=~"unhealthy|healthy"}[$__rate_interval]))',  # noqa: E501
+        ]),
+        legendFormat="State",
+    )],
+    thresholds=[
+        Threshold("green", 0, 95.0),
+        Threshold("orange", 1, 90.0),
+        Threshold("red", 2, 0.0),
+    ],
+)
+
+
+quotaStatusCode = TimeSeries(
+    title="Quota evaluation status code over time",
+    dataSource="${DS_PROMETHEUS}",
+    fillOpacity=30,
+    lineInterpolation="smooth",
+    unit=UNITS.OPS_PER_SEC,
+    targets=[Target(
+        expr='sum by (code) (rate(s3_cloudserver_quota_evaluation_duration_seconds_count{namespace="${namespace}", code=~"2.."}[$__rate_interval]))',  # noqa: E501
+        legendFormat="Success",
```

Comment:
* either you `sum by(code)`, to get one time serie per code → the legendFormat must be based on the code (typically `{{ code }}`)
* or you want a single serie "success" → use a simpler sum, without grouping
* 
```suggestion
        expr='sum(rate(s3_cloudserver_quota_evaluation_duration_seconds_count{namespace="${namespace}", code=~"2.."}[$__rate_interval]))',  # noqa: E501
        legendFormat="Success",
```

Reply:
must also filter by job/service